frontend/javascripts/oxalis/view/action-bar/tracing_actions_view.tsx

```diff
@@ -491,7 +448,7 @@ export const getTracingViewModalsAndMenuItems = (
     });
   }
 
-  menuItems.push(layoutMenu);
+  layoutMenu != null && menuItems.push(layoutMenu);
```

Comment:
I usually avoid such short hands as I find it harder to read than:
```suggestion
  if (layoutMenu != null) {
    menuItems.push(layoutMenu);
  }
```

Reply:
I abbreviated the if-clause by omitting the brackets for it to be in one line, I hope thats okay too :)